src/lorarmesh.cpp

```diff
@@ -14,25 +31,60 @@ LoraMesh::LoraMesh(DataQueue<message_t>* txQueue,
   this->txQueue = txQueue;
   this->rxQueue = rxQueue;
   this->router = router;
+
+  Layer1 = new Layer1Class();
+  Layer1->setPins(LORA_CS, LORA_RST, LORA_IRQ);
+  Layer1->setTxPower(TX_POWER);
+  Layer1->setLoRaFrequency(LORA_FREQ);
+
+  int attempts = MAX_CONNECT_RETRIES;
+
+  while (!Layer1->init()) {
+    delay(CONNECT_DELAY_MILLS);
+    if (attempts-- == 0)
+      ESP.restart();
+  }
+  LL2 = new LL2Class(Layer1);
+  LL2->setInterval(10000);  // set to zero to disable routing packets
```

Comment:
Why 10000? What this value means?

Reply:
10000 is the value of the `_routingInterval variable`, which is used as the data sending interval for LoRaLayer2. It's used in the condition for receiving or transmitting data.

Reply:
We still don't know which value to use. Research is needed to define an appropriate value.

Reply:
I opened issue #36 to investigate that.